src/axom/inlet/Function.cpp

```diff
@@ -40,17 +40,27 @@ Function& Function::registerVerifier(std::function<bool(const Function&)> lambda
   return *this;
 }
 
-bool Function::verify() const
+bool Function::verify(std::vector<VerificationError>* errors) const
```

Comment:
Would it make sense to use an inlet type alias for `std::vector<VerificationError>` ?

Reply:
I'd have to think about it but I'm not sure that an alias over a `std::vector` makes sense here; my concern is that a user would have to look it up (in the Doxygen or elsewhere) to see that it's just an STL type so they know how to iterate over it versus just seeing that it's a `std::vector'`.  Any thoughts @white238 ?

Reply:
I am with @joshessman-llnl on this one. It's a simple STL type and user facing, I would like it clear as possible to them.